### PR TITLE
Add Test\AccessibleObject class

### PR DESCRIPTION
**Pull request by keradus**
In some very edge cases we need to get non-public property/method.
So stop duplicate the code.

### CODE REVIEW

File: Symfony/CS/Test/AccessibleObject.php [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+{
+    private $object;
+    private $reflection;
+
+    public static function create($object)
+    {
+        return new self($object);
+    }
+
+    public function __construct($object)
```
Comment (SpacePossum) [inline Symfony/CS/Test/AccessibleObject.php]: Can't the constructor be `private`? I see no use of this constructor as is..dunno..

Comment (keradus) [inline Symfony/CS/Test/AccessibleObject.php]: Sorry, don't understand. Could you elaborate please?

File: Symfony/CS/Test/AccessibleObject.php [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace Symfony\CS\Test;
+
+/**
+ * @author Dariusz Rumiński <dariusz.ruminski@gmail.com>
+ */
```
Comment (SpacePossum) [inline Symfony/CS/Test/AccessibleObject.php]: Don't you want this class to be marked as `internal` as well? I don't think you want to support uses of this class outside of the project?

File: Symfony/CS/Test/AccessibleObject.php (hunk 1 above)
Comment (SpacePossum) [inline Symfony/CS/Test/AccessibleObject.php]: I was to fast commenting on this, sorry, all good to me :+1: 
Is there a way to type hint `$object` in a PHPDoc as being something of an object (i.e. not scalar)?

File: Symfony/CS/Test/AccessibleObject.php (hunk 2 above)
Comment (keradus) [inline Symfony/CS/Test/AccessibleObject.php]: Personally I have no need for that, it's already final and functionality is so extra small that interface won't change.

Comment (SpacePossum) [inline Symfony/CS/Test/AccessibleObject.php]: very well :)

**Comment by SpacePossum** [comment]
:+1:

File: Symfony/CS/Test/AccessibleObject.php (hunk 1 above)
Comment (keradus) [inline Symfony/CS/Test/AccessibleObject.php]: done

File: Symfony/CS/Test/AccessibleObject.php [hunk 3]
```diff
@@ -0,0 +1,80 @@
...
+        $method = $this->reflection->getMethod($name);
+        $method->setAccessible(true);
+
+        return $method->invokeArgs($this->object, $arguments);
+    }
+
+    public function __isset($name)
+    {
+        try {
+            return isset($this->$name);
```
Comment (stof) [inline Symfony/CS/Test/AccessibleObject.php]: this looks broken to me. this will call ``__isset`` again

Comment (keradus) [inline Symfony/CS/Test/AccessibleObject.php]: Great catch!
I added utests and fix that bug
